app/models/ticket.rb

```diff
@@ -1209,6 +1209,15 @@ def articles
 
   private
 
+  def check_aspect
+    if !aspect
+      self.aspect = "Other"
+    end
+    true
+  end
+
+  private
```

Comment:
I believe private acts until it is explicitly canceled so I believe this private is extra and can be removed.